networking/understanding-networking.adoc

```diff
@@ -12,4 +12,9 @@ IP address means that Pods can be treated like physical hosts or virtual
 machines in terms of port allocation, networking, naming, service discovery,
 load balancing, application configuration, and migration.
 
+[NOTE]
+====
+Use of link-local IP addresses, which are assigned from the `169.254.0.0/16` CIDR block, are supported in {product-title} only for Pods with the `spec.hostNetwork` parameter set to `true`.
```

Comment:
I'm a bit unclear as to the use of "only for Pods" here. I take it to mean that OCP only supports link-local IP addresses when a Pod uses the `spec.hostNetwork` parameter set to `true`. The sentence is also a bit long and hard to follow all the relationships. Suggest breaking into two sentences, maybe something like:

> Link-local IP addresses are assigned from the `169.254.0.0/16` CIDR block. OCP only supports link-local IP addresses that have the `spec.hostNetwork` parameter set to `true`.

^^ Not sure if that's the correct meaning behind this, so finding a way to clarify what the user needs to do with the parameter could be helpful.

Reply:
@bobfuru, `spec.hostNetwork` is a field for the Pod resource. The CIDR block is unreachable by a pod unless that field is set to `true`. That is what I'm trying, with limited success, to explain.

Reply:
That's the message I was getting, @jboxman. It could be helpful to state that more clearly up front, perhaps: 

`For the CIDR block to be reachable by a Pod, the `spec.hostNetwork" field must be set to `true`.

And then fill in the rest? Definitely tricky to shoehorn all the details here.